app/reducers/search.js

```diff
@@ -0,0 +1,23 @@
+import * as searchFilters from '../actions/searchActions';
+
+/**
+ * search criteria
+ *
+ * @param {Object} currentState
+ * @param {String} actionType
+ * @param {Object} payload
+ * @param {String} [payload.filterName]
```

Comment:
I think we should replace `[payload.filterName]` to `payload.filerName`
We use `[var]` for optional parameters
Reference:
http://usejsdoc.org/tags-param.html